cmd/kubeadm/app/phases/copycerts/copycerts.go

```diff
@@ -234,6 +234,8 @@ func DownloadCerts(client clientset.Interface, cfg *kubeadmapi.InitConfiguration
 		return errors.Wrap(err, "error decoding secret data with provided key")
 	}
 
+	fmt.Printf("[download-certs] Download the certificates to the folder: %q\n", cfg.CertificatesDir)
```

Comment:
```suggestion
	fmt.Printf("[download-certs] Saving the certificates to folder: %q\n", cfg.CertificatesDir)
```

Reply:
pls see: https://github.com/kubernetes/kubernetes/pull/113005#discussion_r994069053    😄 

I think both of words are okay to me.

Reply:
Having the 'ing' verb is better here. Saving / Downloading doesn't matter much.

Reply:
Done